src/components/Actions/Actions.css

```diff
@@ -69,8 +63,18 @@
     color: white;
   }
 
+  .operations {
+    flex-grow: 1;
+    justify-content: flex-end;
+  }
+
+  .operations li {
+    &:not(:first-child) {
```

Comment:
There is only one `li` element inside `ul.operations`, it seems like this rule will never be applied.

Reply:
`Clear` button was there alongside `Show` button in the original design. I'm still thinking about bringing it back so I kept the necessary style.